<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[..............Visit MySpace Login URL..............]]>
    </line>
    <line>
      <![CDATA[........Validate Login Header........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Empty UserName and Password........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Empty UserName and Any Password........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Any UserName and Empty Password........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Bad UserName and Bad Password........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Bad UserName and Good Password........]]>
    </line>
    <line>
      <![CDATA[........Validate Login with Good UserName and Good Password........]]>
    </line>
    <line>
      <![CDATA[[Console Log] Screenshot saved in com.web.tests.WebTESTCASES.MySpaceTests_screenshot.png]]>
    </line>
  </reporter-output>
  <suite name="Automation Suite" duration-ms="46426" started-at="2017-01-16T09:39:54Z" finished-at="2017-01-16T09:40:40Z">
    <groups>
      <group name="Regression">
        <method signature="MySpaceTests.Login_Invalid(com.web.tests.myspace.login.LoginDP)[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="Login_Invalid" class="com.web.tests.WebTESTCASES.MySpaceTests"/>
      </group> <!-- Regression -->
    </groups>
    <test name="Web Automation Tests" duration-ms="46426" started-at="2017-01-16T09:39:54Z" finished-at="2017-01-16T09:40:40Z">
      <class name="com.web.tests.WebTESTCASES.MySpaceTests">
        <test-method status="PASS" signature="init()[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="init" is-config="true" duration-ms="150" started-at="2017-01-16T15:09:54Z" finished-at="2017-01-16T15:09:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="testPreRequisites()[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="testPreRequisites" is-config="true" duration-ms="1" started-at="2017-01-16T15:10:02Z" finished-at="2017-01-16T15:10:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPreRequisites -->
        <test-method status="FAIL" signature="Login_Invalid(com.web.tests.myspace.login.LoginDP)[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="Login_Invalid" duration-ms="34542" started-at="2017-01-16T15:10:02Z" data-provider="LoginDP" finished-at="2017-01-16T15:10:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[com.web.tests.myspace.login.LoginDP@1339e7aa]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed due to Invalid Login credentials expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed due to Invalid Login credentials expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.web.tests.myspace.login.Login.inValidCredentials(Login.java:73)
	at com.web.tests.WebTESTCASES.MySpaceTests.Login_Invalid(MySpaceTests.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[..............Visit MySpace Login URL..............]]>
            </line>
            <line>
              <![CDATA[........Validate Login Header........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Empty UserName and Password........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Empty UserName and Any Password........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Any UserName and Empty Password........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Bad UserName and Bad Password........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Bad UserName and Good Password........]]>
            </line>
            <line>
              <![CDATA[........Validate Login with Good UserName and Good Password........]]>
            </line>
            <line>
              <![CDATA[[Console Log] Screenshot saved in com.web.tests.WebTESTCASES.MySpaceTests_screenshot.png]]>
            </line>
          </reporter-output>
        </test-method> <!-- Login_Invalid -->
        <test-method status="PASS" signature="destroy(org.testng.ITestResult)[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="destroy" is-config="true" duration-ms="2757" started-at="2017-01-16T15:10:37Z" finished-at="2017-01-16T15:10:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Login_Invalid status=FAILURE method=MySpaceTests.Login_Invalid(com.web.tests.myspace.login.LoginDP)[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5] output=..............Visit MySpace Login URL..............]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- destroy -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.web.tests.WebTESTCASES.MySpaceTests@3bd82cf5]" name="tearDown" is-config="true" duration-ms="0" started-at="2017-01-16T15:10:40Z" finished-at="2017-01-16T15:10:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.web.tests.WebTESTCASES.MySpaceTests -->
    </test> <!-- Web Automation Tests -->
  </suite> <!-- Automation Suite -->
</testng-results>
